zk-addition-dapp-with-noir-and-nextjs/zk-addition-dapp-with-noir-and-nextjs.md

```diff
@@ -0,0 +1,328 @@
+# ZK-addition-dapp with Noir and Nextjs
+
+![https://static.wixstatic.com/media/6e1afa_d8f9270372b34b4aa443502f98a41aed~mv2.jpg/v1/fill/w_1332,h_697,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/6e1afa_d8f9270372b34b4aa443502f98a41aed~mv2.jpg](https://static.wixstatic.com/media/6e1afa_d8f9270372b34b4aa443502f98a41aed~mv2.jpg/v1/fill/w_1332,h_697,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/6e1afa_d8f9270372b34b4aa443502f98a41aed~mv2.jpg)
+
+We will demonstrate a step-by-step exploration of a basic zk-dapp designed for verifying additions. This application enables users to prove that the sum of two numbers, X and Y, equals Z without disclosing the actual numbers on the blockchain. Although solving this problem does not necessarily demand zero-knowledge proofs, we will utilize them in this example to maintain simplicity and enhance comprehension.
+
+Let’s dive into the code, or [<ins>skip</ins>](https://github.com/RareSkills/blog/blob/prod/zk-addition-dapp-with-noir-and-nextjs/zk-addition-dapp-with-noir-and-nextjs.md#run-dapp) to running the dapp.
+
+We will start by cloning [noir-zk-fullstack-example](https://github.com/RareSkills/noir-zk-fullstack-example.git) repo locally;
+```bash
+git clone https://github.com/RareSkills/noir-zk-fullstack-example.git
+```
+
+>[!NOTE] 
```

Comment:
The note format won't render, I'd make this italic or something

Reply:
on github or elsewhere?